src/utils/registry.ts

```diff
@@ -0,0 +1,272 @@
+  },
+  webkit: {
+    'ubuntu18.04': ['pw_run.sh'],
+    'ubuntu20.04': ['pw_run.sh'],
+    'mac10.13': undefined,
+    'mac10.14': ['pw_run.sh'],
+    'mac10.15': ['pw_run.sh'],
+    'mac11': ['pw_run.sh'],
+    'mac11-arm64': ['pw_run.sh'],
+    'win32': ['Playwright.exe'],
+    'win64': ['Playwright.exe'],
+  },
+  ffmpeg: {
+    'ubuntu18.04': ['ffmpeg-linux'],
+    'ubuntu20.04': ['ffmpeg-linux'],
+    'mac10.13': ['ffmpeg-mac'],
+    'mac10.14': ['ffmpeg-mac'],
+    'mac10.15': ['ffmpeg-mac'],
+    'mac11': ['ffmpeg-mac'],
+    'mac11-arm64': ['ffmpeg-mac'],
+    'win32': ['ffmpeg-win32.exe'],
+    'win64': ['ffmpeg-win64.exe'],
+  },
+};
+
+const DOWNLOAD_URLS = {
+  chromium: {
+    'ubuntu18.04': '%s/builds/chromium/%s/chromium-linux.zip',
+    'ubuntu20.04': '%s/builds/chromium/%s/chromium-linux.zip',
+    'mac10.13': '%s/builds/chromium/%s/chromium-mac.zip',
+    'mac10.14': '%s/builds/chromium/%s/chromium-mac.zip',
+    'mac10.15': '%s/builds/chromium/%s/chromium-mac.zip',
+    'mac11': '%s/builds/chromium/%s/chromium-mac.zip',
+    'mac11-arm64': '%s/builds/chromium/%s/chromium-mac-arm64.zip',
+    'win32': '%s/builds/chromium/%s/chromium-win32.zip',
+    'win64': '%s/builds/chromium/%s/chromium-win64.zip',
+  },
+  firefox: {
+    'ubuntu18.04': '%s/builds/firefox/%s/firefox-ubuntu-18.04.zip',
+    'ubuntu20.04': '%s/builds/firefox/%s/firefox-ubuntu-18.04.zip',
+    'mac10.13': '%s/builds/firefox/%s/firefox-mac-10.14.zip',
+    'mac10.14': '%s/builds/firefox/%s/firefox-mac-10.14.zip',
+    'mac10.15': '%s/builds/firefox/%s/firefox-mac-10.14.zip',
+    'mac11': '%s/builds/firefox/%s/firefox-mac-10.14.zip',
+    'mac11-arm64': '%s/builds/firefox/%s/firefox-mac-11.0-arm64.zip',
+    'win32': '%s/builds/firefox/%s/firefox-win32.zip',
+    'win64': '%s/builds/firefox/%s/firefox-win64.zip',
+  },
+  webkit: {
+    'ubuntu18.04': '%s/builds/webkit/%s/webkit-ubuntu-18.04.zip',
+    'ubuntu20.04': '%s/builds/webkit/%s/webkit-ubuntu-20.04.zip',
+    'mac10.13': undefined,
+    'mac10.14': '%s/builds/webkit/%s/webkit-mac-10.14.zip',
+    'mac10.15': '%s/builds/webkit/%s/webkit-mac-10.15.zip',
+    'mac11': '%s/builds/webkit/%s/webkit-mac-10.15.zip',
+    'mac11-arm64': '%s/builds/webkit/%s/webkit-mac-11.0-arm64.zip',
+    'win32': '%s/builds/webkit/%s/webkit-win64.zip',
+    'win64': '%s/builds/webkit/%s/webkit-win64.zip',
+  },
+  ffmpeg: {
+    'ubuntu18.04': '%s/builds/ffmpeg/%s/ffmpeg-linux.zip',
+    'ubuntu20.04': '%s/builds/ffmpeg/%s/ffmpeg-linux.zip',
+    'mac10.13': '%s/builds/ffmpeg/%s/ffmpeg-mac.zip',
+    'mac10.14': '%s/builds/ffmpeg/%s/ffmpeg-mac.zip',
+    'mac10.15': '%s/builds/ffmpeg/%s/ffmpeg-mac.zip',
+    'mac11': '%s/builds/ffmpeg/%s/ffmpeg-mac.zip',
+    'mac11-arm64': '%s/builds/ffmpeg/%s/ffmpeg-mac.zip',
+    'win32': '%s/builds/ffmpeg/%s/ffmpeg-win32.zip',
+    'win64': '%s/builds/ffmpeg/%s/ffmpeg-win64.zip',
+  },
+};
+
+export const allBrowserNames: BrowserName[] = Object.keys(DOWNLOAD_URLS) as BrowserName[];
```

Comment:
I'd rather move this next to BrowserName definition and hardcode the list.

Reply:
sure